src/components/data-management/MetadataPopover.jsx

```diff
@@ -28,11 +40,7 @@ const MetadataPopover = (props) => {
       deleteEnabled={false}
       value={`Track ${existingMetadata.filter((key) => key.match('Track-')).length + 1}`}
       defaultEditing
-      validationFunc={
-        (value) => value.length > 0
-          && !existingMetadata.map((metadataKey) => metadataKey.toLowerCase())
-            .includes(metadataNameToKey(value.toLowerCase()))
-      }
+      validationFunc={(name) => validateInputs(name, validationChecks, validationParams).isValid}
```

Comment:
Could we validate `metadataKeyToName(metadataNameToKey(name))` instead of just `name`. It is a corner case, but I think that validating `name` will not detect uniqueness for names with multiple blanks.

Reply:
Ah I understand what you mean. Yes, I think that would be a a good idea